apps/web/views.py

```diff
@@ -1,8 +1,29 @@
 from django.shortcuts import render
 
+from rest_framework import permissions, status
+from rest_framework.decorators import (api_view,
+                                       permission_classes,
+                                       throttle_classes,)
+from rest_framework.response import Response
+from rest_framework.throttling import AnonRateThrottle
+
+from .serializers import ContactSerializer
+
 
 def home(request, template_name="index.html"):
     """
     Home Page View
     """
     return render(request, template_name)
+
+
+@throttle_classes([AnonRateThrottle, ])
+@api_view(['POST', ])
+@permission_classes((permissions.AllowAny, ))
+def contact_us(request):
```

Comment:
@RishabhJain2018 Can you add a support so that you capture the name and email from the user who is logged in.